src/main.rs

```diff
@@ -723,4 +725,37 @@ mod ok {
 
         results
     }
+
+    fn has_sysfs_support() -> Vec<OkTestResult> {
+        let fs_res = vec![dev_sev_rw(true), dev_sev_rw(false)];
+
+        fs_res
+    }
+
+    fn dev_sev_rw(read: bool) -> OkTestResult {
```

Comment:
A more clear alternative to a bool called `read` might be to take an `fs::OpenOptions` as an argument. You can also pass other helpful information to parameterize this function as you have. For example, a verb as a &str like "reading" or "writing".